cp3pt0-deployment/common/migrate_singleton.sh

```diff
@@ -81,6 +76,88 @@ function main() {
     success "Migration is completed for Cloud Pak 3.0 Foundational singleton services."
 }
 
+function migrate_license_service_reporter(){
+    title "LSR migration from ibm-cmmon-services to ${LSR_NAMESPACE}"
+
+    local lsr_instances=$("$OC" get IBMLicenseServiceReporter instance -n ibm-common-services --no-headers | wc -l)
+    if [[ lsr_instances -eq 0 ]]; then
+        info "No LSR for migration found in ibm-common-services namespace"
+        return 0
+    fi
+    # Prepare LSR PV/PVC which was decoupled in isolate.sh
+    # delete old LSR CR - PV will stay as during isolate.sh the policy was set to Retain
+    ${OC} delete IBMLicenseServiceReporter instance -n ibm-common-services
```

Comment:
Does it make sense to change the order, i.e. check the PV was labelled, before we remove `IBMLicenseServiceReporter` CR? If for some reason adopter hasn't picked the right order, the data is lost without any recovery.

Reply:
moved check for LSR PV to earlier stage